### DISCUSSION TITLE
Pooling with frend

### DISCUSSION THREAD

**Discussion by tjquan91** [Chia-Network/chia-blockchain#1711] [Q&A]
'Pooling with my friend'
Situation 1
Me and my friend plot and farm using the same account with a total of 20TB plots.
Situation 2
Me and my friend plot and farm using 2 different accounts, but same total of 20TB plots (we divide earnings equally after)

Will situation 1 increase our chance of winning? or is it the same since we have 20TB in both situations.

**Comment by hoffmang9** [comment]
You will have the exact same outcome as the only thing that bears on winning is total space plotted versus total netspace.
